### ISSUE TITLE
TypeError: (intermediate value).toLowerCase is not a function

### ISSUE THREAD

**Issue by maxpower9000** [Jabher/cornerjs#12]
**Issue**
Error message inside Firefox console:  
`TypeError: (intermediate value).toLowerCase is not a function`.

The message appears on every pageload. Independent of any use of `directive()`.  
Just because the browser is loading the `corner.js` or `corner.min.js`.

The responsible line of code is:
https://github.com/Jabher/cornerjs/blob/1f4ffd17e3abd10b7150cc3be612bd82e84ecb37/build/corner.js#L748

**My setup**  
`corner.js 0.9.3` installed through Yarn
Firefox Developer Edition 60.0b5 on Win7 32bit WAMP-Server

**Comment by maxpower9000** [comment]
**Chrome 64 says:**
```js
Uncaught TypeError: (e.className || "").toLowerCase is not a function
    at element_loaded (corner.min.js:1)
    at element_loaded_wrapper (corner.min.js:1)
    at NodeList.forEach (<anonymous>)
    at itar (corner.min.js:1)
    at corner.min.js:1
    at Array.forEach (<anonymous>)
    at itar (corner.min.js:1)
    at MutationObserver.observe.childList (corner.min.js:1)
```

**Comment by Jabher** [comment]
Hello! Is it happening in some exact page/domain or all across the internet?

**Comment by maxpower9000** [comment]
@Jabher Only tested on localhost.

I found some similar issues on SO:
- https://stackoverflow.com/a/42036382
- https://stackoverflow.com/a/25412298

Maybe the issue here also has to do with the **automatic semicolon insertion** ...

>The ECMAScript specification has specific rules for automatic semicolon insertion, however in this case a semicolon isn't automatically inserted because the parenthesised expression that begins on the next line can be interpreted as an argument list for a function call.
> This means that without that semicolon, the anonymous window.Glog function was being invoked with a function as the msg parameter, followed by (window) which was subsequently attempting to invoke whatever was returned.

**Comment by Jabher** [comment]
No, it's different. Can you please catch an error with the debugger and show what inside an "e"?
It should be some kind of HTMLElement.

Or, you can create some test page so I can debug it.

**Comment by jrussnone** [comment]
I'm having this issue in a project as well. I found that if an SVG is parsed, the property className is actually an object not a string. I have a potential fix in #14
